app/helpers/samples_helper.rb

```diff
@@ -587,16 +589,9 @@ def self.samples_by_metadata_field(sample_ids, field_name)
           ] + Location::GEO_LEVELS.map { |l| "#{l.pluralize}_locations.name" }
         )
     else
-      # TODO(jsheu): Modify to get validated field from MetadataField.
-      first_datum = Metadatum.where(key: field_name).first
-      if first_datum
-        query
-          .includes(:metadata_field)
-          .group(first_datum.validated_field)
```

Comment:
^ This was one recent hacky example. Shouldn't need to fetch the Metadatum just to call `validated_field`